File: C/stopwatch.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    int n, s = 0;
    scanf("%d", &n);

    if (n % 2 != 0) {
        printf("still running\n");
        return 0;
    }

    for (int i = 0; i < n; i+= 2) {
        int a, b;
        scanf("%d %d", &a, &b);
        s += abs(a - b);
    }


    printf("%d\n", s);
}

File: Kotlin/stopwatch.kt
import kotlin.math.*

fun main() {
    val n = readLine()!!.toInt()

    if (n % 2 != 0) {
        println("still running")
        return
    }

    var sum = 0
    for (i in 0..(n-1)/2) {
        sum += abs(readLine()!!.toInt() - readLine()!!.toInt())
    }

    println(sum)
}